bin/fillTestData/Users.js

```diff
@@ -202,7 +202,7 @@ var addUsers = function () {
                 // non admin user
                   user.email = index + faker.internet.email();
                   user.password = faker.internet.password();
-                  user.username = index + user.displayName.toLowerCase().replace('\'', '').replace(' ', '');
```

Comment:
replace without regexp replaces only the first occurrence of the searched string, not all of them.